tests/phpunit/test-class-wc-stripe-upe-payment-method.php

```diff
@@ -317,4 +352,23 @@ public function test_payment_methods_are_reusable_if_cart_contains_subscription(
 			}
 		}
 	}
+
+	/**
+	 * Test the type of payment token created for the user.
+	 */
+	public function test_create_payment_token_for_user() {
+		$user_id = 1;
+
+		foreach ( $this->mock_payment_methods as $payment_method_id => $payment_method ) {
+			if ( 'card' === $payment_method_id ) {
+				$card_payment_method_mock = $this->array_to_object( self::MOCK_CARD_PAYMENT_METHOD_TEMPLATE );
+				$token = $payment_method->create_payment_token_for_user( $user_id, $card_payment_method_mock );
+				$this->assertTrue( 'WC_Payment_Token_CC' === get_class( $token ) );
```

Comment:
```suggestion
				$this->assertTrue( 'WC_Payment_Token_CC' === get_class( $token ) );
				$this->assertEquals( 'pm_mock_payment_method_id', $token->get_token() );
				$this->assertEquals( '4242', $token->get_last4() );
```
Think there might be any value in adding a few more assertions to check the properties on the token?

Reply:
Yeah, I can do that. Which properties are you thinking?

Reply:
I was just thinking of checking `get_token()` to ensure it stores the ID and `get_last4()` to check the credential. I don't think there's really anything else that important.

Reply:
```suggestion
				$this->assertEquals( 'WC_Payment_Token_CC', get_class( $token ) );
```
This could be rewritten as the above, though `assertEquals` doesn't check strict types...I'm happy to forget I ever noticed this.